src/app/pages/HomePage/HomePage.tsx

```diff
@@ -60,15 +58,14 @@ const HomePage = ({ pageData }: HomePageProps) => {
         lang={lang}
         description={description}
         openGraphType="website"
-        hasAmpPage
+        hasAmpPage={false}
       />
       <LinkedData
         type="CollectionPage"
         seoTitle={title}
         headline={title}
         entities={[itemList]}
       />
-      <Ad slotType="leaderboard" />
```

Comment:
Oh dear, this seems to have been removed by accident (which is causing some of the tests to fail). Can we put this back 🙏 ?